### DISCUSSION TITLE
Graph interaction with commits

### DISCUSSION THREAD

**Discussion by rodhash** [SuperBo/fugit2.nvim#104] [General]
When in Graph view, can we select and open the desired commit hash to read it?

**Comment by SuperBo** [comment]
What you need to read, commit hash or commit diff changes?

If you want to get commit hash, you can use key `y` to copy hash string.

**Reply by rodhash** [reply]
Reading some long commits messages with multiple lines. Or reading the code changed by the commit itself.

It seems hovering the commit and hitting Enter doesn't bring me more details about the commit, right?

**Reply by SuperBo** [reply]
yes, you can create a feature request for it, so that I can track what I need to do. :D
